skyportal/handlers/photometry.py

```diff
@@ -15,8 +20,58 @@ def post(self):
         description: Upload photometry
         parameters:
           - in: path
-            name: photometry
-            schema: Photometry
+            name: time_format
```

Comment:
These fields still need descriptions.  No idea what are valid values for "time_format"!

Reply:
Done